src/openbsdsoftraid_fmt_plug.c

```diff
@@ -97,10 +96,13 @@ static struct custom_salt {
 static void init(struct fmt_main *self)
 {
 #ifdef _OPENMP
-	omp_t = omp_get_max_threads();
-	self->params.min_keys_per_crypt *= omp_t;
-	omp_t *= OMP_SCALE;
-	self->params.max_keys_per_crypt *= omp_t;
+	int omp_t = omp_get_max_threads();
+
+	if (omp_t > 1) {
+		self->params.min_keys_per_crypt *= omp_t;
+		omp_t *= OMP_SCALE;
+		self->params.max_keys_per_crypt *= omp_t;
+	}
```

Comment:
Somehow no syntax highlighting here - maybe unbalanced braces somewhere before?

Reply:
The code seems to be fine. I checked it again, and couldn't spot anything problematic.